### ISSUE TITLE
crio --help prints redundant or contradicting default values

### ISSUE THREAD

**Issue by dilyanpalauzov** [cri-o/cri-o#6213]
### What happened?

`crio --help` prints for some parameters twice the default value.  In case of `--drop-infra-ctr`, `-enable-criu-support `, and `--selinux`  the two default values differ.  For `--log-journald `, `--no-pivot`, `--read-only` and `--stream-enable-tls` the same default value is printed twice.

The help string for `--read-only` uses inconsisting quoting.  Moreover I do not understand why does it print `/run` as in `   --read-only /run    Setup all unprivileged …`.

```console
crio --help
VERSION:
   1.25.0
Version:        1.25.0
GitCommit:      f36bc747908c1609efcce490e0d16c785f5372c0
GitCommitDate:  2022-08-30T09:57:07Z
GitTreeState:   clean
BuildDate:      2022-08-30T16:53:57Z
GoVersion:      go1.18.5
Compiler:       gc
Platform:       linux/amd64
Linkmode:       dynamic
BuildTags:      
  containers_image_ostree_stub
  exclude_graphdriver_btrfs
  btrfs_noversion
  containers_image_openpgp
  seccomp
  selinux
LDFlags:          unknown
SeccompEnabled:   true
AppArmorEnabled:  false
Dependencies:     
…
 --drop-infra-ctr                         Determines whether pods are created without an infra container, when the pod is not using a pod level PID namespace (default: true) (default: false) [$CONTAINER_DROP_INFRA_CTR]
 --enable-criu-support                    Enable CRIU integration, requires that the criu binary is available in $PATH. (default: '') (default: false) [$CONTAINER_ENABLE_CRIU_SUPPORT]
 --log-journald                           Log to systemd journal (journald) in addition to kubernetes log file (default: false) (default: false) [$CONTAINER_LOG_JOURNALD]
 --no-pivot pivot_root                    If true, the runtime will not use pivot_root, but instead use `MS_MOVE` (default: false) (default: false) [$CONTAINER_NO_PIVOT]
 --read-only /run                         Setup all unprivileged containers to run as read-only. Automatically mounts tmpfs on /run, `/tmp` and `/var/tmp`. (default: false) (default: false) [$CONTAINER_READ_ONLY]
 --selinux                                Enable selinux support (default: true) (default: false) [$CONTAINER_SELINUX]
 --stream-enable-tls                      Enable encrypted TLS transport of the stream server (default: false) (default: false) [$CONTAINER_ENABLE_TLS]

```

### What did you expect to happen?

Not applicable.

### How can we reproduce it (as minimally and precisely as possible)?

Type
```console
crio --help
```

### CRI-O and Kubernetes version

<details>

```console
$ crio --version
crio version 1.25.0
Version:        1.25.0
GitCommit:      f36bc747908c1609efcce490e0d16c785f5372c0
GitCommitDate:  2022-08-30T09:57:07Z
GitTreeState:   clean
BuildDate:      2022-08-30T16:53:57Z
GoVersion:      go1.18.5
Compiler:       gc
Platform:       linux/amd64
Linkmode:       dynamic
BuildTags:      
  containers_image_ostree_stub
  exclude_graphdriver_btrfs
  btrfs_noversion
  containers_image_openpgp
  seccomp
  selinux
LDFlags:          unknown
SeccompEnabled:   true
AppArmorEnabled:  false
Dependencies:     
```

</details>

### OS version

<details>

```console
$ cat /etc/os-release
NAME="Fedora Linux"
VERSION="36 (Workstation Edition)"
ID=fedora
VERSION_ID=36
VERSION_CODENAME=""
PLATFORM_ID="platform:f36"
PRETTY_NAME="Fedora Linux 36 (Workstation Edition)"
ANSI_COLOR="0;38;2;60;110;180"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:36"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f36/system-administrators-guide/"
SUPPORT_URL="https://ask.fedoraproject.org/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=36
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=36
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Workstation Edition"
VARIANT_ID=workstation

# paste output here
$ uname -a
Linux d 5.19.6-200.fc36.x86_64 #1 SMP PREEMPT_DYNAMIC Wed Aug 31 17:58:15 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

</details>

### Additional environment details (AWS, VirtualBox, physical, etc.)

Not applicable.

**Comment by dilyanpalauzov** [comment]
On the release-1.25 branch this problem is still present.

**Comment by haircommander** [comment]
will be fixed in 1.25 in https://github.com/cri-o/cri-o/pull/6229 :)
